rabbitmq/src/main/java/io/micronaut/rabbitmq/intercept/RabbitMQConsumerAdvice.java

```diff
@@ -112,6 +127,37 @@ public RabbitMQConsumerAdvice(BeanContext beanContext,
         }
     }
 
+    /**
+     * Deprecated constructor.
+     *
+     * @param beanContext       The bean context
+     * @param binderRegistry    The registry to bind arguments to the method
+     * @param exceptionHandler  The exception handler to use if the consumer isn't a handler
+     * @param serDesRegistry    The serialization/deserialization registry
+     * @param conversionService The service to convert consume argument values
+     * @param channelPools      The channel pools to retrieve channels
+     * @deprecated
```

Comment:
tell users in the `@deprecated` java doc to use the other constructor instead with `{@link`